src/main/java/com/gluonhq/substrate/model/ReleaseConfiguration.java

```diff
@@ -27,16 +27,18 @@
  */
 package com.gluonhq.substrate.model;
 
+import java.util.Objects;
+
 public class ReleaseConfiguration {
 
     public static final String DEFAULT_BUNDLE_VERSION = "1.0";
-    public static final String DEFAULT_BUNDLE_SHORT_VERSION = "1.0";
     public static final String DEFAULT_MAC_APP_CATEGORY = "public.app-category.utilities";
 
-    public static final String DEFAULT_CODE_VERSION = "1";
-    public static final String DEFAULT_CODE_NAME = "1.0";
-
-    public static final String DEFAULT_APP_VERSION = "1.0.0";
+    private static final String DEFAULT_APP_DESCRIPTION = "Default description";
+    private static final String DEFAULT_BUNDLE_SHORT_VERSION = "1.0";
+    private static final String DEFAULT_CODE_VERSION = "1";
```

Comment:
Not sure why we named this "default_code_version" but the field is `versionCode`. Since we are at it, can you rename it to `DEFAULT_VERSION_CODE`?